### ISSUE TITLE
Is Populate supposed to work?

### ISSUE THREAD

**Issue by Andrejs1979** [arqo123/speedgoose#90]
It's sometimes working, sometimes not. 

User.findOne({ providerID: authSession.user_id }).populate(
      "accounts._account"
    ),
    .cacheQuery()

If I remove cacheQuery(), then populate works every time.

**Comment by arqo123** [comment]
Yes, populate supose to work normally. Could you please bring me some more examples or reproduce steps?

**Comment by arqo123** [comment]
closing due to unactivity, and not being able to reproduce it :( Please reopen it if needed.
